app/build.gradle.kts

```diff
@@ -137,11 +137,14 @@ dependencies {
     implementation("androidx.compose.ui:ui-tooling:${Versions.COMPOSE_VERSION}")
     implementation("androidx.compose.foundation:foundation:${Versions.COMPOSE_VERSION}")
     implementation("androidx.compose.material:material:${Versions.COMPOSE_VERSION}")
+    implementation("com.google.accompanist:accompanist-systemuicontroller:${Versions.COMPOSE_ACCOMPANIST_VERSION}")
```

Comment:
We already have `androidx.compose.foundation:foundation` why do we have to add this 🙏

Reply:
@Wadeewee we need to use `rememberSystemUiController` 🤔